rules/opentitan_test.bzl

```diff
@@ -321,6 +323,7 @@ def opentitan_functest(
             DEFAULT_BITSTREAM = {
                 "cw310_test_rom": "@//hw/bitstream:test_rom",
                 "cw310_rom": "@//hw/bitstream:rom",
+                "cw310_other": "@//hw/bitstream:rom",
```

Comment:
I think we'd rather `fail()` than grab the //hw/bitstream:rom for "cw310_other", right?

Reply:
Ack, I think I can fix this. (The default bitstream is looked up and then overwritten so I'll need to construct a dummy here that fails if we use it.)

Reply:
I was wrong, I only needed that before I fixed something else in this file. Thanks!